src/app/components/add-order/add-order.component.ts

```diff
@@ -75,29 +89,55 @@ export class AddOrderComponent implements OnInit {
       typeOfChargeID: ['', Validators.required],
       typeOfReceiptID: ['', Validators.required],
       orderStatusID: [1],
-      sellerID:["9ef10519-9e30-471b-ad2b-f6f1798cfd15"],
+      sellerID:["8a5a6095-9b31-4693-9fc1-8a53c29a3e23"],
```

Comment:
Use the **seller service** in the UnitOfWork to load data conditionally **if the user is an admin or employee** in a select option element in the document.